docs/source/configuring_zipkin.rst

```diff
@@ -84,6 +84,16 @@ zipkin.blacklisted_routes
         'zipkin.blacklisted_routes': ['some_internal_route',]
 
 
+zipkin.host
+~~~~~~~~~~~~~~~~~~
+    The host ip that is used for zipkin spans.
```

Comment:
The wording here makes it look like they're required. Can you put some statement that says something about how if they're not given, then the host/port will be automatically determined?